devicetypes/fibargroup/fibaro-walli-double-switch.src/child-metering-switch.groovy

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Child Metering Switch", namespace: "fibargroup", author: "SmartThings", mnmn: "SmartThings", vid: "generic-switch-power-energy") {
```

Comment:
As far as I know you could still have just used the smartthings-namespace version of this DTH?

Reply:
@greens @dkirker 
You recommended moving this (parent) DTH to the "fibargroup" namespace.
After that it was noticed that child device wasn't created (reported in ICP-12783)

that's how  adding a child device looks in a parent DTH:
`addChildDevice("Child Metering Switch", childDni, device.getHub().getId(), [
				completedSetup	: true,
				label			: componentLabel,
				isComponent		: false
			])`

Is there another way (syntax) to reach out for the Child Metering Switch (that is in smartthings namespace already) ? If so - then copying the child-metering-switch will not be needed on the "fibargroup" side.

Reply:
Ah. Yeah just looked at the underlying code. Could you use `fibaro-double-switch-2-usb`?

Reply:
@greens We might try that, but fibaro-double-switch-2-usb is defined in a bit different namespace "**FibarGroup**". Will it work without changing it ?
Suggestion: Maybe all those fibaro dths should be unified to work under the same namespace ? That would avoid confusions.

Reply:
My hunch is that the namespace is case-insensitive, but test it out. The main thing we want to avoid is having to create two DTHs here, so if you can't get that to work, go ahead and move this back to the smartthings namespace.

Reply:
@greens 
If we want to stick to "fibaro-double-switch-2-usb" (who doesn't send its DNI when on/off is called) then:
1)  both parent "Fibaro Double Switch 2 ZW5" and child "fibaro-double-switch-2-usb"
2)  or  parent "Fibaro Walli Double Switch" (who handles child devices in more generic way right now)

 should be refactored.

Reply:
UPDATE: both parent DTHs were refactored, a child DTH was refactored too.